client/router.coffee

```diff
@@ -48,7 +44,7 @@ Router.route "/admins",
   onBeforeAction: ->
     redirectIfNotAuthorized(@, ['admin'])
   waitOn: ->
-    Meteor.subscribe "allUsers"
+    [Meteor.subscribe("allUsers"), Meteor.subscribe("roles")]
```

Comment:
Ohh, good point. I was looking to make things consistent but probably not worth the effort.
Let's leave it.